apps/protocol-frontend/src/components/ProfileDaos.tsx

```diff
@@ -0,0 +1,116 @@
+import { Button, Flex, Heading, Divider, SimpleGrid } from '@chakra-ui/react';
+import { ControlledSelect, GovrnSpinner } from '@govrn/protocol-ui';
+import { useDaosList } from '../hooks/useDaosList';
+import DaoCard from './DaoCard';
+
+//TODO: add the Join function
+
+// only here for mocking purposes, will replace
+type DaoRoles = 'admin' | 'contributor' | 'recruit';
+
+type Dao = {
+  id: number;
+  name: string;
+  role: DaoRoles;
+  favorite: boolean;
+};
```

Comment:
Some code duplication here, with dao card

Reply:
Oh yeah -- figure once we remove the mockData that'll use the same type in both places (likely our UIGuild type)